public/sass/elements/forms/_form-multiple-choice.scss

```diff
@@ -8,19 +8,15 @@
   clear: left;
   position: relative;
 
-  padding: (8px $gutter-one-third 9px 50px);
+  padding: 0 0 0 38px;
   margin-bottom: $gutter-one-third;
 
-  cursor: pointer; // Encourage clicking on the label
-
    // remove 300ms pause on mobile
   -ms-touch-action: manipulation;
   touch-action: manipulation;
 
   @include media(tablet) {
     float: left;
-    padding-top: 7px;
-    padding-bottom: 7px;
     // width: 25%; - Test : check that text within labels will wrap
```

Comment:
I wonder if we can just dump this comment with the other two lines. It’s been [hanging around since 2014](https://github.com/alphagov/govuk_elements/commit/7835339e070063d566e98df2162b725cc59c379d).